.github/workflows/mayhem.yml

```diff
@@ -0,0 +1,112 @@
+
```

Comment:
Could you please use a matrix action to run these analyses instead? For example https://github.com/mayhemheroes/mayhem-cmake-example/blob/solution/.github/workflows/mayhem.yml#L61-L62

Reply:
Just made the required changes. Will shortly update my other submissions to use this mayhem.yml format as well